ext/magic/extconf.rb

```diff
@@ -19,7 +19,7 @@
 
 unless File.exist?(filename)
   File.open(filename, 'wb') do |target_file|
-    URI.open("https://fossies.org/linux/misc/file-#{LIBMAGIC_TAG}.tar.gz", 'rb') do |read_file|
+    URI.open("https://fossies.org/linux/misc/file-#{LIBMAGIC_TAG}.tar.gz", "User-Agent" => "RubyMagic") do |read_file|
```

Comment:
@dosuken123 - Should we use `RUBY_DESCRIPTION` constant instead as the agent is a Ruby script?

Also according to [RFC-7231](https://tools.ietf.org/html/rfc7231#section-5.5.3), this header field SHOULD be sent which means it's recommended but not mandatory so IMHO the implementation on FOSSIES side is wrong. Should we inform them about this as well?

Reply:
>  IMHO the implementation on FOSSIES side is wrong

Not mandatory does not mean that the server may not insist. We might request they drop this recondition, but I can understand the reasons for having it.

Reply:
`RUBY_DESCRIPTION` is quite a mouthful. 😄 

```
irb(main):001:0> RUBY_DESCRIPTION
=> "ruby 2.7.2p137 (2020-10-01 revision 5445e04352) [x86_64-darwin19]"
```

Reply:
How about we use this instead:
```suggestion
    URI.open("https://fossies.org/linux/misc/file-#{LIBMAGIC_TAG}.tar.gz", "User-Agent" => "RubyMagic/#{RUBY_DESCRIPTION}") do |read_file|
```

this loosely follows: 

```
     User-Agent = product *( RWS ( product / comment ) )
```

as per their example:

```
     User-Agent: CERN-LineMode/2.15 libwww/2.17b3
```

On my machine this is what I get:

```
"RubyMagic/ruby 2.7.2p137 (2020-10-01 revision 5445e04352) [x86_64-darwin20]"
```

Reply:
```suggestion
    URI.open("https://fossies.org/linux/misc/file-#{LIBMAGIC_TAG}.tar.gz", "User-Agent" => "RubyMagic/#{RUBY_DESCRIPTION}") do |read_file|
```